src/lib/utils/functions.ts

```diff
@@ -51,8 +51,14 @@ export function processData<T> (data: T, _default: T, process?: (T) => T): T | u
 }
 
 export function handleTermSignals (io): void {
+  if (realtime.state === State.Starting) {
+    process.exit(0)
+  } else if (realtime.state === State.Stopping) {
```

Comment:
Let's split this into two independent ifs as the else block makes it just harder to read :)

Reply:
Done